vnext/Microsoft.ReactNative/Views/ExpressionAnimationStore.h

```diff
@@ -14,8 +14,8 @@ namespace uwp {
 // resolved, so they can be reused.
 class ExpressionAnimationStore {
  public:
-  comp::ExpressionAnimation GetElementCenterPointExpression();
-  comp::ExpressionAnimation GetTransformCenteringExpression();
+  comp::ExpressionAnimation GetElementCenterPointExpression(comp::Compositor compositor);
```

Comment:
This PR should probably be marked with breakng change..

Reply:
@dannyvv  why's that? no interfaces are being changed AFAICT.